# viRNA: A collection of conserved viral RNA structures

Michael T. Wolfinger<sup>1,2</sup>

<sub><sup>1</sup>Department of Theoretical Chemistry, University of Vienna, Vienna, Austria</sub><br/>
<sub><sup>2</sup>Research Group Bioinformatics and Computational Biology, Faculty of Computer Science, University of Vienna, Vienna, Austria</sub>


This repository is intended to provide a (not necessarily complete) collection of structural multiple sequence alignments of evolutionary conserved RNA structures in viruses. Currently, specific ecologic groups of flaviviruses are covered.

Each alignment of viral sequences represents a particular RNA family of structurally homologous entities and has been built either from known RNA families or derived de-novo by comparative genomics. All alignments are in Stockholm format, which allows easy construction of [Infernal](http://eddylab.org/infernal/) covariance models.

While some of the alignments are also available though the [Rfam](https://rfam.xfam.org) database, several alignments provided here have been specifically built for particular viral species or groups. This allows for a fine-grained characterization and comparison of conserved RNA elements in different viruses, as outlined in the below publications.

If you use data provided in this repository in your own work please link back to this site and cite the following publications - Thank you!

- **Functional RNA Structures in the 3â€™UTR of Mosquito-Borne Flaviviruses**  
 _Michael T. Wolfinger, Roman Ochsenreiter, Ivo L. Hofacker_  
 (2021) [Preprint PDF](https://github.com/mtw/viRNA/blob/main/Publications/Wolfinger-2021__PREPRINT.pdf) | [doi:10.1201/9781003097679-5](https://dx.doi.org/10.1201/9781003097679-5)

- **Functional RNA Structures in the 3'UTR of Tick-Borne, Insect-Specific and No-Known-Vector Flaviviruses**  
 _Roman Ochsenreiter, Ivo L. Hofacker, Michael T. Wolfinger_  
 Viruses 11(3):298 (2019) [PDF](http://lindor.tbi.univie.ac.at/fileadmin/media/papers/Ochsenreiter-2019.pdf) | [doi:10.3390/v11030298](https://doi.org/10.3390/v11030298)
